scripts/devops_tasks/setup_execute_tests.py

```diff
@@ -307,3 +335,5 @@ def execute_global_install_and_test(
         execute_global_install_and_test(args, targeted_packages, extended_pytest_args)
     else:
         prep_and_run_tox(targeted_packages, args, extended_pytest_args)
+
+    collect_log_files()
```

Comment:
Am concerned. 

`execute_tox_serial` in `tox_harness.py:277` will blow away the tox directory after it finishes up with each run. This is because the CI machines _do not_ have very much space and a full venv is pretty big size wise.

Gonna need to exclude it from the cleanup over there.

Reply:
Locally you probably haven't seen this because you're running in a way that doesn't trigger the `in_ci` bool.

To try in pshell, call `$Env:TF_BUILD=whatever_text_you_want_needs_to_be_non_empty`

Reply:
I believe I have worked around this now